GlyssenEngineTests/Character/Template/CharacterAssignerTests.cs

```diff
@@ -1,8 +1,12 @@
-﻿using System;
+﻿/// <summary>
```

Comment:
CharacterAssignerTests should not have been moved into Template. They should still be set to compile

Reply:
Moved. Sorry, I got sloppy.